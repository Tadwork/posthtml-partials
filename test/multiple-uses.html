<partial name="say" class from>
	<p class="{{ class }}">Hello from {{ from }}!</p>
</partial>

<div>
	<partial name="say" class="hello" from="me"></partial>
	<partial name="say" class="bye" from="him"></partial>
</div>
