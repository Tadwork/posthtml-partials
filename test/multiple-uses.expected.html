

<div>
	
	<p class="hello">Hello from me!</p>

	
	<p class="bye">Hello from him!</p>

</div>
